app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,33 @@
+class MicropostsController < ApplicationController
+  before_action :logged_in_user, only: %i(index create destroy)
+  before_action :correct_user, only: :destroy
+
+  def create
+    @micropost = current_user.microposts.build micropost_params
+    if @micropost&.save
+      flash[:success] = t ".micropost_created"
+      redirect_to root_path
+    else
+      #redirect_to root_path
+      @feed_items = current_user.feed.paginate page: params[:page]
+      render "static_pages/home"
+    end
+  end
+
+  def destroy
+    @micropost.destroy
```

Comment:
if `destroy` will return fail result, this project will display message `success`.
what is this yours expectancy?